packages/MSBot/src/msbot.ts

```diff
@@ -7,17 +7,17 @@ import * as chalk from 'chalk';
 import * as program from 'commander';
 import * as process from 'process';
 
-const pkg = require('../package.json');
+const pkg: IPackage = require('../package.json');
 const semver = require('semver');
```

Comment:
'typedef' warning not fixed, this line has already been modified in [#16](https://github.com/southworkscom/botbuilder-tools/pull/16) for 'no-var-requires' rule.